README.md

```diff
@@ -1 +1,50 @@
-# Math-Magician
\ No newline at end of file
+# Math-Magicians
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- React
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- HTML
+- CSS
+- JavaScript
+- React
+- Git
+- GitHub
+
+### Install
+
+To clone this repo go to "https://github.com/Donard97/math-magician"
+
+
+## Author
+
+👤 **Donard Golaj**
+
+- Github : [@Donard97](https://github.com/Donard97)
+- Twitter: [@donardgolaj](https://twitter.com/donardgolaj)
+- LinkdIn : [@donard-golaj](https://www.linkedin.com/in/donard-golaj/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Donard97/math-magician/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] [OPTIONAL] Kindly let's fix the path to the license file, for now, it seems to be broken.